test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": "00"
+    },
+    {
+      "dec": "1",
+      "hex": "01"
+    },
+    {
+      "dec": "-1",
+      "hex": "ff"
```

Comment:
It is the twos complement, that is negative.
On 11 Apr 2015 04:28, "Fedor Indutny" <notifications@github.com> wrote:

> In test/fixtures/bn.json
> <https://github.com/indutny/bn.js/pull/36#discussion_r28168842>:
>
> > @@ -0,0 +1,105 @@
> > +{
> > +  "valid": [
> > +    {
> > +      "dec": "0",
> > +      "hex": "00"
> > +    },
> > +    {
> > +      "dec": "1",
> > +      "hex": "01"
> > +    },
> > +    {
> > +      "dec": "-1",
> > +      "hex": "ff"
>
> Why should this happen? Hex could be negative too, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/indutny/bn.js/pull/36/files#r28168842>.
>